examples/numalogic-simple-pipeline/ml_steps/utility.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import os
+from dataclasses import dataclass
+import mlflow
+from dataclasses_json import dataclass_json
+from numalogic.registry import MLflowRegistrar
+from numpy._typing import ArrayLike
+
+DIR = os.path.dirname(__file__)
+ROOT_DIR = os.path.split(DIR)[0]
+TRAIN_DATA_PATH = os.path.join(ROOT_DIR, "ml_steps/resources/train_data/train_data.csv")
+TRACKING_URI = "http://mlflow-service.numaflow-system.svc.cluster.local:5000"
+LOGGER = logging.getLogger(__name__)
+
+
+@dataclass_json
+@dataclass
+class Payload:
+    data: ArrayLike = None
+    anomaly_score: float = 0.0
+    uuid: str = None
```

Comment:
@vigith do you think we should have this dataclass in numalogic?

Reply:
we could call it as timeseries data?